tensorboard/plugins/graph/tf_graph_controls/tf-graph-controls.ts

```diff
@@ -1079,9 +1086,14 @@ class TfGraphControls extends LegacyElementMixin(PolymerElement) {
     notify: true,
   })
   traceInputs: boolean = false;
+  @property({
+    type: Boolean,
+    notify: true,
+  })
+  autoExtractNodes: boolean = false;
```

Comment:
For now, I'd prefer preserving today's existing default behavior and keep `autoExtractNodes = true` by default. In the future, we can always change the default easily, based on user feedback. Would that be reasonable?

Reply:
Oops. Sorry. Leftover from testing.